lib/Reader/reader.cpp

```diff
@@ -6564,6 +6569,7 @@ void ReaderBase::readBytesForFlowGraphNodeHelper(
       goto GEN_BRANCH;
 
     GEN_BRANCH:
+      nop();
```

Comment:
Would love to see a comment here as to why we need a `nop`, even if it will just be there in a debug compile. I think I know why...maybe...